perfkitbenchmarker/network.py

```diff
@@ -125,6 +129,25 @@ def GetNetwork(cls, vm):
     """
     return cls.GetNetworkFromNetworkSpec(cls._GetNetworkSpecFromVm(vm))
 
+  @staticmethod
+  def FormatCidrString(cidr_raw):
+    """ Format CIDR for use in resource name.
+
+    eg '10.128.0.0/9' -> '10-128-0-0-9'
+
+    Args:
+      cidr_raw: The unformatted CIDR string.
+    """
+
+    DELIM = r'-'  # Safe delimiter for most providers
+    int_regex = r'[0-9]+'
+    try:
+      return DELIM.join(regex_util.ExtractAllMatches(int_regex, str(cidr_raw)))
+    except:
```

Comment:
Prefer to explicitly fail if it fails validation.

http://google.github.io/styleguide/pyguide.html#244-decision
Never use catch-all except: statements, or catch Exception or StandardError

Reply:
Fixed - raises error now. 
And thank you for the style guide link... should be added to CONTRIBUTING.md